config.rb

```diff
@@ -90,6 +90,51 @@
       path_prefix: "services/teaching-vacancies",
     ),
   },
+  {
+    title: "Get Into Teaching application documentation",
+    pages: GitHubRepoFetcher.instance.docs(
+      service_name: "Get Into Teaching",
+      repo_name: "DFE-Digital/get-into-teaching-app",
+      path_in_repo: "doc",
+      path_prefix: "services/get-into-teaching",
+    ),
+  },
+  {
+    title: "Get Into Teaching API documentation",
+    pages: GitHubRepoFetcher.instance.docs(
+      service_name: "Get Into Teaching",
+      repo_name: "DFE-Digital/get-into-teaching-api",
+      path_in_repo: "docs",
+      path_prefix: "services/get-into-teaching",
+    ),
+  },
+  {
+    title: "Get Teacher Training Adviser documentation",
+    pages: GitHubRepoFetcher.instance.docs(
+      service_name: "Get Into Teaching",
+      repo_name: "DFE-Digital/get-teacher-training-adviser-service",
+      path_in_repo: "docs",
+      path_prefix: "services/get-into-teaching",
+    ),
+  },
+  {
+    title: "Get Into Teaching Asset Manager",
+    pages: GitHubRepoFetcher.instance.docs(
+      service_name: "Get Into Teaching",
+      repo_name: "DFE-Digital/GITISContent",
+      path_in_repo: "docs",
+      path_prefix: "services/get-into-teaching",
+    ),
+  },
+  {
+    title: "Get School Experience documentation",
+    pages: GitHubRepoFetcher.instance.docs(
+      service_name: "School Experience",
```

Comment:
Should this be "Get School Experience"? So it's consistent (ie app-level, rather than team-level) on the left hand side

<img width="294" alt="Screenshot 2021-11-08 at 11 50 17" src="https://user-images.githubusercontent.com/642279/140737092-d6509754-4929-438b-9a9a-b7b6c01390e1.png">

Reply:
Yes, Agreed and changed